### PR TITLE
Included definition for LCBench and improve handling of datasets

### PR DESCRIPTION
**Pull request by mseeger**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: benchmarking/definitions/definition_lcbench.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    "shuttle",
+    "sylvine",
+    "vehicle",
+    "volkert",
+]
+
+
+# This configuration space requires a surrogate to be used. This is needed
+# anyway, since the blackbox is not densely evaluated on a grid
+_config_space = {
```
Comment (geoalgo) [inline benchmarking/definitions/definition_lcbench.py:60]: Could we import the config-space defined in lcbench rather to avoid the duplication?

**Review by geoalgo — COMMENTED** [review]
LGTM, just have a small comment.

File: benchmarking/definitions/definition_lcbench.py (hunk 1 above)
Comment (mseeger) [reply benchmarking/definitions/definition_lcbench.py:60]: OK.

Comment (mseeger) [reply benchmarking/definitions/definition_lcbench.py]: OK, done. I also added a fix

**Review by geoalgo — APPROVED** [review]

**Pull request merged**